### DISCUSSION TITLE
What is: Map Source = Offline vector maps

### DISCUSSION THREAD

**Discussion by velle** [osmandapp/OsmAnd#17184] [Q&A]
When setting up a specific Map Profile, eg "Driving", I go to "Configure Map". 

There I can choose the menu item Map Source, and here I can choose a few different, one of them being "Offline vector maps".

I understand what vector maps mean and what offline means. But exactly which / what offline vector data is it? 

Is it an OSM-file (or osm.pbf)?

Is it one specific OBF-file? A number of OBF-files? Is it some database inside the OsmAnd app? Can this database be said to correspond to a specific OBF file?

Sincerely

**Comment by vshcherb** [comment]
Offline vector maps means the maps you download in Download Maps section
